apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -108,9 +108,13 @@ export class FilesStorageUC {
 	}
 
 	private createPath(schoolId: EntityId, fileRecordId: EntityId): string {
-		const pathToFile = [schoolId, fileRecordId].join('/');
+		try {
+			const pathToFile = [schoolId, fileRecordId].join('/');
 
-		return pathToFile;
+			return pathToFile;
+		} catch (err) {
+			throw new BadRequestException(err, 'FilesStorageUC:createPath');
+		}
```

Comment:
This is the only line that are not catch possible js errors before they leave the uc. I do not think that we can run in this situation, but in the uc method it self we have no try catch because this is only orchtestration. For the line it is maybe inpossible that they throw an error but if they throw an error we run in problems for the error output validation.

I think we found a better solution for it by outsourcing the private code and i remove the throw for now. 

But please keep in mind for my changes the same line with same error is catch in one of the higer methods. I move it to this place because on other places it catch map and re-throw other errors.